cms/djangoapps/contentstore/views/helpers.py

```diff
@@ -291,3 +291,18 @@ def is_item_in_course_tree(item):
         ancestor = ancestor.get_parent()
 
     return ancestor is not None
+
+
+def get_course_hash_value(course_key):
+    """
+    Returns a hash value for the given course key.
+
+    If course key is None, function returns an out of bound value which will
+    never satisfy the vem_enabled_courses_percentage condition
+    """
+    out_of_bound_value = 100
```

Comment:
It is just added as a precaution. 
I am checking it here, so it made sense to cater for the case in case course key is none in hash function as well. https://github.com/edx/edx-platform/blob/4396a30904d3595896fc005d943a54f89cdd5d7c/cms/djangoapps/contentstore/views/videos.py#L832